src/main/resources/META-INF/services/com.github.tomakehurst.wiremock.jetty.JettyHttpServerFactoryLoader

```diff
@@ -0,0 +1,2 @@
+com.github.tomakehurst.wiremock.jetty11.Jetty11HttpServerFactoryLoader
```

Comment:
Quick comment on why Jetty 11 and Jetty 12 bundled in the same service descriptor. Java uses lazy initialization of service loader providers, Jetty 11 comes first and as such, will be tried first.